src/pages/workspaces/workspace/analysis/AnalysisLauncher.js

```diff
@@ -264,7 +265,7 @@ const PreviewHeader = ({
   const analysisLink = Nav.getLink(analysisLauncherTabName, { namespace, name, analysisName })
   const isAzureWorkspace = !!workspace.azureContext
   const currentRuntimeToolLabel = getToolLabelFromRuntime(runtime)
-  const enableJupyterLabPersistenceId = `${namespace}/${name}/enableJupyterLabGCP`
+  const enableJupyterLabPersistenceId = `${namespace}/${name}/${ENABLE_JUPYTERLAB_ID}`
```

Comment:
Hmm... I could be misunderstanding what you're saying, a bit confused since the feature ID is `JUPYTERLAB_GCP_FEATURE_ID`, while the per-workspace-storage key uses uniquely-identifiable workspace info appended with `ENABLE_JUPYTERLAB_ID`. They should be separate variables, and the latter string used here is not used anywhere but in this fashion.

Reply:
`ENABLE_JUPYTERLAB_ID` is an internal ID for a feature. As such, in theory, we could change its value at will without any user-visible impact. But by linking this local storage key to it, now we've added restrictions to it. With this, changing the feature ID would clear this setting for all users. Practically, it may not be an issue since `ENABLE_JUPYTERLAB_ID` is unlikely to change. But generally, often a little duplication is better than creating a dependency between components.